frontend/src/pages/Download.tsx

```diff
@@ -0,0 +1,98 @@
+import { useNavigate } from "react-router-dom";
+import { useEffect, useState } from "react";
+import Modal from "../components/Modal";
+import Button from "../components/Button";
+
+export function Download() {
+  const navigate = useNavigate();
+  const [countdown, setCountdown] = useState(5);
+  const [copied, setCopied] = useState("");
+
+  const startDownload = () => {
+    const link = document.createElement("a");
+    link.href = "https://testcqserver.ngrok.dev/5095_modified.7z";
+
+    document.body.appendChild(link);
+    link.click();
+    document.body.removeChild(link);
+
+    console.log("started download");
+  };
+  const downloadAdobe = () => {
+    const link = document.createElement("a");
+    link.href = "https://testcqserver.ngrok.dev/FPPSetup.exe";
+
+    document.body.appendChild(link);
+    link.click();
+    document.body.removeChild(link);
+
+    console.log("download adobe");
+  };
+  const copyLink = () => {
+    navigator.clipboard.writeText(window.location.href);
+    setCopied("Copied!");
+    console.log("copy link");
```

Comment:
Tu powinna być zmiana tekstu w buttonie na `Copied to Clipboard` i button disabled